data-transforms/ts-converter/README.adoc

```diff
@@ -0,0 +1,163 @@
+= Convert Timestamps using Rust
+:page-layout: lab
+:env-docker: true
+:page-categories: Development, Stream Processing, Data Transforms
+:description: Convert timestamps from various forms, such as epochs to strings.
+// Set up attributes to hold the latest version of Redpanda and Redpanda Console.
+// For GitHub, hard-code the latest version to these values:
+ifndef::env-site[]
+:latest-redpanda-version: 24.1.2
+:latest-console-version: 2.5.2
+endif::[]
+// For the docs site, use the built-in attributes that store the latest version as fetched from GitHub releases.
+ifdef::env-site[]
+:latest-redpanda-version: {full-version}
+// All pages already have access to {latest-console-version} on the docs site.
+endif::[]
+// ========================AUTOMATED TESTS===================================
+// The comments in this file are used to run automated tests of all the documented steps. Tests are run on each pull request to the upstream repository using GitHub Actions. For more details about the testing tool we use, see https://doc-detective.com/.
+
+// (test start {"id":"data-transform-ts-convert-rust", "description": "Convert timestamps from various forms, such as epochs to strings.", "setup": "doc-tests/fetch-versions-and-rpk.json", "cleanup": "doc-tests/cleanup.json"})
+
+This lab contains an example transform written in Rust and using the Redpanda Schema Registry. It allows for transforming keys or values with timestamps in a variety of formats to different formats. For example, converting numeric epoch values in milliseconds to string-based output (or vice-versa).
+
+== Prerequisites
+
+You must have the following:
+
+- At least version 1.75 of https://rustup.rs/[Rust^] installed on your host machine.
+- The Wasm target for Rust installed. To install this target, run the following:
++
+[source,bash]
+----
+rustup target add wasm32-wasi
+----
+// (step {"action":"runShell", "command": "rustup target add wasm32-wasi"})
+- link:https://docs.redpanda.com/current/get-started/rpk-install/[Install `rpk`] on your host machine.
+- https://docs.docker.com/compose/install/[Docker and Docker Compose] installed on your host machine.
```

Comment:
```suggestion
- https://docs.docker.com/compose/install/[Docker and Docker Compose^] installed on your host machine.
```

This notation makes the link open a new tab. It's standard in our docs to do this for external links.